COMETwebapp/Components/DxChart/ChartBarSeries.razor

```diff
@@ -0,0 +1,64 @@
+﻿<!------------------------------------------------------------------------------
```

Comment:
i would not make a folder or namespace called DxCharts. The fact that we use DevExpress is not relevant and i don;t think it should be part of the name. Either invent a new name that says something about why and how the controls or used, or delete the DxChart folder and move all controls to the Components folder.